langchain/embeddings/vertexai.py

```diff
@@ -0,0 +1,33 @@
+"""Wrapper around Google VertexAI embedding models."""
+from typing import Dict, List, Optional
+
+from pydantic import root_validator
+
+from langchain.embeddings.base import Embeddings
+from langchain.llms.vertexai import _VertexAICommon
+from langchain.utilities.vertexai import raise_vertex_import_error
+
+
+class VertexAIEmbeddings(_VertexAICommon, Embeddings):
+    model_name: Optional[str] = "textembedding-gecko@001"
```

Comment:
@lkuligin similar change here. Consider setting to `textembedding-gecko` for latest version. WDYT?

Reply:
done